jekyll/assets/css/main.scss

```diff
@@ -130,6 +132,17 @@ footer {
 	}
 }
 
+div.sidebar-offset {
+  margin-left: $leftbar-width;
+  display: flex;
+  height: 100vh;
+  overflow: scroll;
+
+	@media screen and (max-width: $screen-sm) {
```

Comment:
Indentation is weird here... looks like tabs when the rest of the file uses two spaces.